Solutions/Corvus.UriTemplate.TavisApi.Tests/UriTemplateExtensionsTests.cs

```diff
@@ -252,7 +252,55 @@ public void PartiallyParametersFromAnObjectFromInvalidUrl()
             })
             .Resolve();
 
-            Assert.Equal("http://dev.example.org/v2/customers{?active,country}", url);
+            Assert.Equal("http://dev.example.org/v2/customers{?active}{&country}", url);
```

Comment:
I think, these should not change when nothing has been resolved.

Reply:
In an ideal world, that would be the case, but it would require multiple passes over the string and/or allocating a list to achieve that, and we are prioritising perf (providing it remains functionally equivalent).

I will open an enhancement issue for this.

Reply:
https://github.com/corvus-dotnet/Corvus.UriTemplates/issues/82